### PR TITLE
[Editor] : Presentation stabilization

### PR DESCRIPTION
**Pull request by cmoinier**
This PR fixes various UI issues on the dashboard in the metadata editor.

- Manage singular/plural translations for several keys
- Update French translations
- Change landing page title

### CODE REVIEW

File: translations/fr.json [hunk 1]
```diff
@@ -24,12 +24,12 @@
...
   "dashboard.records.noRecord": "Aucun jeu de données pour cette organisation",
   "dashboard.records.noUser": "Aucun utilisateur pour cette organisation",
-  "dashboard.records.publishedRecords": "données publiées",
+  "dashboard.records.publishedRecords": "{count, plural, =1{donnée publiée} other{données publiées}}",
   "dashboard.records.search": "Résultats pour \"{searchText}\"",
   "dashboard.records.userDetail": "Nom",
   "dashboard.records.userEmail": "Email",
   "dashboard.records.username": "Nom d'utilisateur",
-  "dashboard.records.users": "utilisateurs",
+  "dashboard.records.users": "{count, plural, =1{utlisateur} other{utilisateurs}}",
```
Comment (Angi-Kinas) [inline translations/fr.json:32]:
```suggestion
  "dashboard.records.users": "{count, plural, =1{utilisateur} other{utilisateurs}}",
```

File: libs/ui/elements/src/lib/metadata-info/metadata-info.component.html [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+    translate
+  >
+    record.metadata.about
+  </p>
+  <div *ngIf="metadata.status" class="flex items-start pt-3 pr-4">
+    <span
+      class="inline-flex items-center justify-center px-2 py-1 text-13 uppercase font-medium leading-none bg-primary text-white rounded opacity-40"
+      translate
+    >
+      domain.record.status.{{ metadata.status }}
```
Comment (Angi-Kinas) [inline libs/ui/elements/src/lib/metadata-info/metadata-info.component.html:13]: Should this be part of this PR? I'm wondering where this is being displayed.

File: translations/fr.json (hunk 1 above)
Comment (cmoinier) [reply translations/fr.json:32]: Good catch, thanks!

File: libs/ui/elements/src/lib/metadata-info/metadata-info.component.html (hunk 2 above)
Comment (cmoinier) [reply libs/ui/elements/src/lib/metadata-info/metadata-info.component.html:13]: Oh, that must be something from my other PR that slipped in. Will remove it, thanks!

**Review by Angi-Kinas — APPROVED** [review]

**Pull request merged**